File: test7C#/test7/Model/Donut/BostonDonut.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace test7.Model.Donut
{
    class BostonDonut : SimpleDonut
    {
        public BostonDonut() : base("Boston")
        {
            base.Flavor("Boston Cream");
        }
    }
}

File: test7Go/oop/models/donut/BostonDonut.go
package donut

type BostonDonut struct {
	SimpleDonut
}

func NewBostonDonut() BostonDonut {
	simpleDonut := NewSimpleDonut("Boston")
	simpleDonut.Flavor = "Boston Cream"

	return BostonDonut{simpleDonut}
}
